src/main/webapp/script.js

```diff
@@ -0,0 +1,17 @@
+//implementing map
+var map;
+function initMap() {
```

Comment:
I would suggest putting this in a separate js file. businesses.js. It will reduce merge conflicts in future

Reply:
done